vep/src/main/resources/listVariants.py

```diff
@@ -24,21 +24,25 @@ def main():
     """
     print('Python version: %s' % platform.python_version())
 
-    # get the source and output directories
-    dataset_srcdir = f'{S3DIR}/variants/*/*/*'
-    ld_server_srcdir = f'{S3DIR}/ld_server/variants/*'
-    outdir = f'{S3DIR}/out/varianteffect/variants'
+    # just specify, potentially with a list only new datasets
+    dataset_srcdir = f'{S3DIR}/variants/ExSeq/*/*'
+    existing_variants = f'{S3DIR}/out/varianteffect/variants'
+    new_variants = f'{S3DIR}/out/new-variants'
```

Comment:
This should probably still be in `out/varianteffect/new-variants` not in the first level of out, I imagine this is just an error carried from when you were writing to the test bucket

Reply:
Sounds good.  I mean to call out for your opinion on the directories.